blog/content/post/2020-12-09-OSFC_2020.md

```diff
@@ -0,0 +1,198 @@
+---
+title: Thoughts around OSFC 2020
+abstract: 'Let's share some thoughts that evolved during the OSFC 2020 talks and send kudos to many people who made this conference happen.'
+cover: /covers/OSFC.png
+author: kamila.banecka
+layout: post
+published: true
+date: 2020-12-09
+archives: "2020"
+
+tags:
+  - event
+  - OpenSource
+  - fimware
+categories:
+  - Firmware
+  - Security
+
+---
+This year we've had double track insight into the [Open Source Firmware
+Conference](https://osfc.io/) as both participants and presenters. At the
+beginning big kudos to 9elements for organizing the event and gather crucial
+voices of contributors from various firmware community corners. We want to share
+some thoughts that evolved during the talks and send kudos to many people who
+made this conference happen. We will leave a few notes on every attended talk,
+from day 1 to day 3, according to the
+[schedule](https://cfp.osfc.io/osfc2020/schedule/), trying to gather talks with
+references and sources. You can go through the post or skip to the chosen
+conference. Remember, presented descriptions are only our interpretation, not
+official statements of presenters. To watch the full video and get your own
+opinion, click the chosen title.  
+
+### Day 1
+
+* [Why LVFS is Essential for OEMs](https://vimeo.com/488137809) by [Richard Hughes](https://twitter.com/hughsient)
+
+The Linux Vendor Firmware Service, an essential part of the Linux ecosystem, is
+worth digging in. It allows its users to keep hardware up to date with firmware
+updates. Now we are waiting for fwupd for QubesOS. Kudos to [Red Hat](https://2019.osfc.io/uploads/talk/paper/9/Introducing_the_Linux_vendor_Firmware_service.pdf) for your work.
+
+* [UEFI compliance testing](https://vimeo.com/488138727) by Heinrich Schuchardt
+
+These guys are doing great stuff for ARM and RISC-V with tools that are worthy
+of mentioning, e.g., Platform Initialization SCT and Firmware Test Suite (FWTS).
+The overall quality of UEFI-compliance test results is impressive. We liked the
+idea of UEFI Self Certification Tests, UEFI SCT compliance across various UEFI
+implementations for different platforms (QEMU, x86, RISC-V). Kudos to [U-Boot](https://github.com/u-boot/u-boot) for your work.
```

Comment:
@KBanecka this paragraph needs complete rewrite

Reply:
@miczyg1 prośba o pomoc, tutaj są notes:

```
UEFI compliance testing

presenter is wrong about number of UEFI implmentations, we probably should have a blog post about UEFI implmentations available on the market
they have they own test tools
definitely they doing stuff for ARM and RISC-V, since this is how presenter show UEFI Lifetime
TAT should use UEFI SCT - this should be port of Dasharo Compatibility Module
TAT: UEFI SCT compliance across various UEFI implementations for various platforms (QEMU, x86, RISC-V)
we need quality UEFI-complicance test results/reports since definitely market miss that and Denx and other companies see the value of such support
there is also PI-SCT, FWTS
```